### ISSUE TITLE
I work well app iOS on version 7.4.0. When i update on version 7.50, 7.6.0, can't not build Archive or Build iOS

### ISSUE THREAD

**Issue by tuemaytinh1981** [Baseflow/flutter-geolocator#829]
## 🔙 Regression

I tried many ways clearing cache, repairing pod...but it didn't work. What's wrong?. Thank you!
Unable to integrate package geolocator_apple 1.1.0 or 1.20 ???.
On Simulator works fine (Emphasis works fine on emulator) but when build Archive or Build iOS produces an error.

### Old (and correct) behavior
Good integration with previous version 7.4.0.

### Current behavior
/Users/maitrongtue/Documents/Projects/homehunting/ios/Runner/GeneratedPluginRegistrant.m:40:9: fatal error: module 'geolocator_apple' not found
    @import geolocator_apple;
     ~~~~~~~^~~~~~~~~~~~~~~~
    1 error generated.

### Reproduction steps
Just updating to a version higher than 7.4.0 and creating the above error cannot be fixed.
### Configuration
7.4.0 up 7.5.0, 7.6.0
**Version:** 7.5.0, 7.6.0

**Platform:** 
- [x ] :iphone: iOS
- [ ] :robot: Android

**Comment by tuemaytinh1981** [comment]
<img width="1148" alt="Screen Shot 2021-08-30 at 22 21 54" src="https://user-images.githubusercontent.com/41774672/131363193-1455f48e-9559-4683-8b6f-c9fccc860a5f.png">

**Comment by mvanbeusekom** [comment]
Hi @tuemaytinh1981, 

I am trying to reproduce this issue but failing. Could you provide the output of the `flutter doctor -v` command?

**Comment by tuemaytinh1981** [comment]
Thanks @mvanbeusekom !
I found the cause: although podfile in ios:platform :ios, '12.0' but Deployment Target 9.0 in XCode, I set it back to 12.1. Then delete all files in ~/Library/Developer/Xcode/DerivedData. Run Flutter Clean, Pod deintegrate. And I rebuild on Xcode successfully. This is a problem "GeneratedPluginRegistrant" when building on XCode.
Thank you so much for your help.
<img width="752" alt="Screen Shot 2021-08-31 at 10 49 24" src="https://user-images.githubusercontent.com/41774672/131440021-027425d0-a765-4a5a-82a6-f04ae6810f9d.png">
Fix on Xcode:
<img width="517" alt="Screen Shot 2021-08-31 at 10 47 08" src="https://user-images.githubusercontent.com/41774672/131440043-fe42e17d-cc77-4466-9509-cd775d898a28.png">
